magma/circuit.py

```diff
@@ -23,7 +23,8 @@
 from magma.syntax.sequential import sequential
 from magma.syntax.combinational import combinational
 from magma.syntax.sequential import sequential
-from magma.syntax.verilog import combinational_to_verilog, sequential_to_verilog
+from magma.syntax.verilog import combinational_to_verilog, \
+    sequential_to_verilog, build_kratos_debug_info
```

Comment:
why is `build_kratos_debug_info` exposed in `magma.circuit`?

Reply:
I have no idea where to expose.

Reply:
Probably leaving it in `magma.syntax` or `magma.syntax.verilog`

Reply:
I think `magma` doesn't have `syntax` imported and that's why I put in a different place. I will add `syntax` to the top level.